lua/ui/game/build_templates.lua

```
@@ -6,85 +6,85 @@
-        import("/lua/ui/game/announcement.lua").CreateAnnouncement(LOC('<LOC template_0000>Build Template Received'), tab, LOCF('<LOC template_0001>From %s', sender))
-    end
-    AddTemplate(msg.data)
-end
+---@type UIBuildTemplateData[]
+local templates = Prefs.GetFromCurrentProfile('build_templates') or {}
 
+---@param template UIBuildTemplate
+---@return string
 function GetInitialName(template)
     for _, entry in template do
         if type(entry) ~= 'table' then continue end
```

Luacheck (1.2.0)

[error] expected '=' near 'end'

(E011)